pinot-common/src/main/java/org/apache/pinot/common/utils/request/RequestUtils.java

```diff
@@ -178,6 +178,12 @@ public static Expression getLiteralExpression(byte[] value) {
     return expression;
   }
 
+  public static Expression getLiteralExpression(com.google.protobuf.ByteString value) {
```

Comment:
Is this correct? Shouldn't this be calcite `ByteString`?

Reply:
This is used by proto ByteString for v2 request serde.

Reply:
I see. Should we import it, or is it intentional to keep the full class path?

Reply:
we can import this.